nixos/modules/programs/gpu-screen-recorder.nix

```diff
@@ -34,6 +40,36 @@ in {
       capabilities = "cap_sys_nice+ep";
```

Comment:
I mentioned this in another ongoing pull request but I forgot to mention here. Can you remove this cap_sys_nice section? This breaks desktop portal capture. It was commented out in gpu screen recorder installer script too explaining why: https://git.dec05eba.com/gpu-screen-recorder/tree/extra/meson_post_install.sh#n7

Reply:
I couldn't mark it in github before but I think you need to remove the whole `security.wrappers."gpu-screen-recorder" = {` section, not just this line.